woocommerce/woocommerce
# [Settings] Add Section tabs

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes https://github.com/woocommerce/woocommerce/issues/52277

Adds tabs for pages with sections. See LSwxXIhP15A1YvZWnkgihH-fi=350-48825 for designs

<img width="1250" alt="image" src="https://github.com/user-attachments/assets/f3a8a61d-82ee-4d16-88ac-638366b54652">




### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Install/Activate Gutenberg
2. Turn on `settings` feature flag
3. WooCommerce > Settings
4. Make sure url, tabs, and content react as you would expect



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

Adds section tabs to Modern Settings

</details>
